frontend/device/kobo/nickel_conf.lua

```diff
@@ -53,9 +53,7 @@ function NickelConf.frontLightLevel.get()
         -- FrontLightState config, so don't normalize the value here yet.
         return tonumber(new_intensity)
     else
-        -- In NickelConfSpec, require("device") won't return KoboDevice
-        local powerd = require("device/kobo/powerd")
-        local fallback_fl_level = powerd.fl_intensity or 1
+        local fallback_fl_level = 20
```

Comment:
Sorry, that's my fault. This change is in the general stuff that already happens. Apparently there's currently a fallback of a light level of `1` somewhere for Kobo and @Hzj-jie changed it to `20`. It's unrelated to the main stuff of this PR.